### DISCUSSION TITLE
Sotetseg Orb Help?

### DISCUSSION THREAD

**Discussion by Haftago** [runelite/runelite#15210] [Q&A] [answered]
One of the most amazing things about previous clients was that at sotetseg, for people that struggle more for seeing colors. that they had the ability to label the sotetseg orbs, more specifically you had the ability to see when an orb that was coming diagonally and travelling THROUGH Sotetseg. 

In the current version of RL and as OSRS as a whole, those orbs are only visible for 2 ticks (1.2 seconds) before it goes through sotetseg. This gives me 1.2 seconds to both figure out what color of gray it is, and notice that it is coming at me. 

This is doable yes; and I'm not asking for hats or for each orb to be color coded to make it braindead. 

I'm hoping that even just having some sort of outline of the orb (colored or not) to make it so I have more than 2 ticks (1.2 seconds) to be able adjust my prayers accordingly. 

This could be in the form of highlight like: 

1: Inventory tags outline, colorless but can be seen through sotetseg. 
2: this can be an img file inside the ball indicating a orb is present. 
3: this could a hull outline, colorless but can be seen through sotetseg

I'm not asking to make it easier to distinguish range from mage just to reitterate. I'm asking for diagonal orbs to be more distinguishable and not invisble, it's already hard enough being color blind.

**Comment by Nightfirecat** [comment, marked as answer]
Per [Jagex's latest statement on Third Party Clients](https://secure.runescape.com/m=news/third-party-client-guidelines?oldschool=1), we can't offer that feature as it falls under this line item:

> * Projectile targets, or target locations, as well as impact locations
